app/locales/en/translations.coffee

```diff
@@ -86,6 +86,7 @@ I18nTranslationsEn =
     "inbox":
       "quick_links": "Quick Links"
       "all_offers": "All Offers"
+      "create_offer": "Create offer"
       "notifications": "Notifications"
       "new_offers": "New"
```

Comment:
can we remove these coffee files, and add js files, just like we did for stock app @patrixr ?

Reply:
good point 👍